src/coreclr/jit/emitarm64.h

```diff
@@ -164,6 +168,13 @@ inline bool OptimizeLdrStr(instruction ins,
         return true;
     }
 
+    // If we have a second LDR instruction from the same source, then try to replace it with a MOV.
+    if (IsOptimizableLdrToMov(ins, reg1, reg2, imm, size, fmt))
```

Comment:
I'm wondering if this creates a GC hole? Should this be gated by a `!localVar` check, or even better emitIns_Mov variant that handles varx/offs.

Reply:
Also the gcstress pipelines are clean.